content/en/docs/reference/controllers/daemonset.md

```diff
@@ -0,0 +1,106 @@
+---
+title: DaemonSet controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The DaemonSet controller manages Pods configured via a
+{{< glossary_tooltip term_id="daemonset" >}} object, to make sure that some
+(or all) Nodes run a copy of a Pod.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+The DaemonSet controller is built in to kube-controller-manager.
+
+## Controller behavior
+
+The DaemonSet controller ensures that all eligible nodes run a copy of a Pod,
+by creating Pods with a Node assignment (one Pod for each eligible Node).
+
+This controller creates one Pod for each Node that the DaemonSet matches.
+It creates new Pods in batches, doubling the batch size each time a request
+succeeds. This helps to avoid sending a very large number of API requests thatxi
+will ultimately all fail with the same error.
```

Comment:
- s/thatxi/that
- `This helps to avoid sending a very large number of API requests thatxi
will ultimately all fail with the same error`.  This could be reworded; the text seems to imply that a large number of API requests will ultimately fail for some reason.
- Future Pod creation requests are batched after the successful creation of an initial Pod/Node?

Reply:
> a large number of API requests will ultimately fail for some reason.